### PR TITLE
Supporting sequences list by asset id and easier copy for SequenceData and Datapoints

### PR DESCRIPTION
**Pull request by sanderland**
- changed sequences list to go to advanced endpoint and added assetIds
- support for insert sequencedata in sequencedataapi.insert
- support for insert datapoints in datapointsapi.insert

### CODE REVIEW

File: cognite/client/_api/sequences.py [hunk 1]
```diff
@@ -284,8 +289,8 @@ def insert(
             column_ids (Optional[List[int]]): List of ids for the columns of the sequence.
                 If 'None' is passed to both column_ids and columns_external_ids, all columns ids will be retrieved from metadata and used in that order.
             column_external_ids (Optional[List[str]]): List of external id for the columns of the sequence.
-            rows (Union[ Dict[int, List[Union[int, float, str]]], List[Tuple[int,Union[int, float, str]]], List[Dict[str,Any]]]):  The rows you wish to insert.
-                Can either be a list of tuples, a list of {"rowNumber":... ,"values": ...} objects or a dictionary of rowNumber: data. See examples below.
+            rows (Union[ Dict[int, List[Union[int, float, str]]], List[Tuple[int,Union[int, float, str]]], List[Dict[str,Any]], SequenceData]):  The rows you wish to insert.
+                Can either be a list of tuples, a list of {"rowNumber":... ,"values": ...} objects a dictionary of rowNumber: data, or a SequenceData object. See examples below.
```
Comment (erlendvollset) [inline cognite/client/_api/sequences.py]:
```suggestion
                Can either be a list of tuples, a list of {"rowNumber":... ,"values": ...} objects, a dictionary of rowNumber: data, or a SequenceData object. See examples below.
```

File: cognite/client/_api/sequences.py [hunk 2]
```diff
@@ -314,8 +319,20 @@ def insert(
...
+
+            Finally, they can be a SequenceData object retrieved from another request. In this case column_external_ids from this object are used as well.
+
+                >>> from cognite.client.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> data = c.sequences.data.retrieve(id=2,start=0,end=10)
+                >>> c.sequences.data.insert(rows=data, id=1)
         """
         utils._auxiliary.assert_exactly_one_of_id_or_external_id(id, external_id)
+        if isinstance(rows, SequenceData):
```
Comment (erlendvollset) [inline cognite/client/_api/sequences.py]: Perhaps something to implement for datapoints.insert() aswell?

File: cognite/client/_api/sequences.py [hunk 3]
```diff
@@ -135,7 +136,8 @@ def list(self, limit: int = 25) -> SequenceList:
                 >>> for seq_list in c.sequences(chunk_size=2500):
                 ...     seq_list # do something with the sequences
         """
-        return self._list(method="GET", filter=None, limit=limit)
+        filter = {"assetIds": asset_ids if asset_ids else None}
```
Comment (erlendvollset) [inline cognite/client/_api/sequences.py]: unnecessary if-else clause?

Comment (sanderland) [reply cognite/client/_api/sequences.py]: yep

File: cognite/client/_api/sequences.py (hunk 2 above)
Comment (sanderland) [reply cognite/client/_api/sequences.py]: yes, I was thinking about it, but it's a bit trickier with DatapointsList and so on. I'll add just Datapoints

File: cognite/client/_api/datapoints.py [hunk 4]
```diff
@@ -232,14 +232,23 @@ def insert(
...
+            Or they can be a Datapoints object::
+
+                >>> from cognite.client import CogniteClient
+                >>> c = CogniteClient()
+                >>> data = c.datapoints.retrieve(external_id="abc",start=datetime(2018,1,1),end=datetime(2018,2,2))
+                >>> c.datapoints.insert(data, external_id="def")
         """
         utils._auxiliary.assert_exactly_one_of_id_or_external_id(id, external_id)
         post_dps_object = self._process_ids(id, external_id, wrap_ids=True)[0]
+        if isinstance(datapoints, Datapoints):
```
Comment (erlendvollset) [inline cognite/client/_api/datapoints.py]: Add a unit test for this?

Comment (erlendvollset) [reply cognite/client/_api/datapoints.py]: Ah you made the copy integration test. nvm.

**Pull request merged**